Mailer/app/api/v1/Handlers/Users/UserPreferencesHandler.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Users;
+
+use Nette\Utils\Json;
+use Remp\MailerModule\Repository\UserSubscriptionsRepository;
+use Remp\MailerModule\Repository\UserSubscriptionVariantsRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Params\InputParam;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+
+class UserPreferencesHandler extends BaseHandler
+{
+    private $userSubscriptionsRepository;
+
+    private $userSubscriptionVariantsRepository;
+
+    public function __construct(
+        UserSubscriptionsRepository $userSubscriptionsRepository,
+        UserSubscriptionVariantsRepository $userSubscriptionVariantsRepository
+    ) {
+        parent::__construct();
+        $this->userSubscriptionsRepository = $userSubscriptionsRepository;
+        $this->userSubscriptionVariantsRepository = $userSubscriptionVariantsRepository;
+    }
+
+    public function params()
+    {
+        return [
+            new InputParam(InputParam::TYPE_POST, 'user_id', InputParam::REQUIRED),
```

Comment:
Ask for both ID and email until we decide which of those two we'll keep.
_(it will probably be email)_